src/features/people/peopleList/PersonTile/styled.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
```

Comment:
This is very similar to Wrapper from MovieTile, have you tried to add another attribute that would just apply the styles that are different?

Reply:
we will discuss this